ansible_collections/arista/avd/molecule/evpn_underlay_rfc5549_overlay_ebgp/intended/structured_configs/DC1-BL1A.yml

```diff
@@ -337,6 +342,16 @@ ethernet_interfaces:
   shutdown: false
   mtu: 1500
   ipv6_enable: true
+- name: Ethernet11
+  peer: outside-r1
+  peer_interface: other1
+  peer_type: other
+  description: P2P_LINK_TO_outside-r1_other1
+  type: routed
+  shutdown: false
+  mtu: 1500
+  ip_address: 10.23.23.1/30
+  ipv6_enable: true
```

Comment:
You have set ipv6_enable specifically to False. That should not even be necessary. Instead add your extra condition for `routing_protocol != "ebgp"` to the place where we generate ipv6 based on 5549. (It should still be possible to generate ipv6_enable if specifically set to true)

Reply:
Got it. I'm on vacation this week. I'll commit the change to the PR next week.

Reply:
I have put this in draft for now. Set as ready once you have updated. Thanks.

Reply:
Done. Please review.